### PR TITLE
Disable vstest tests temporarily to unblock PR build

### PR DESCRIPTION
**Pull request by codito**

### CODE REVIEW

File: test/dotnet-vstest.Tests/VSTestTests.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Microsoft.DotNet.Cli.VSTest.Tests
 {
     public class VSTestTests : TestBase
     {
-        [Fact]
+        //[Fact]
```
Comment (livarcocc) [inline test/dotnet-vstest.Tests/VSTestTests.cs]: Use Skip

**Comment by livarcocc** [comment]
@dotnet-bot Test CentOS7.1 x64 Debug Build
@dotnet-bot Test RHEL7.2 x64 Release Build
@dotnet-bot Test Windows_NT x64 Release Build
@dotnet-bot Test Windows_NT x86 Debug Build

**Comment by codito** [comment]
> Expected command to pass but it did not.\r\nFile Name: D:\\j\\workspace\\release_windo---4c3b899b\\/artifacts/win81-x64/stage2\\dotnet.exe\r\nArguments: restore

Are the test assets in an incorrect format? Restore seems to fail.

**Comment by livarcocc** [comment]
I think this is a race condition when running some tests that all write to the same location.

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x64 Release Build
@dotnet-bot Test Windows_NT x86 Debug Build

**Pull request merged**